string name
database_map_server/BasePoint[] points